src/legacy/core_plugins/console/public/np_ready/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -66,18 +63,18 @@ function EditorOutputUI() {
   useEffect(() => {
     if (data) {
       const mode = modeForContentType(data[0].response.contentType);
-      editorInstanceRef.current.session.setMode(mode);
-      editorInstanceRef.current.update(
+      editorInstanceRef.current!.session.setMode(mode);
```

Comment:
Would it be worth assigning this to a variable at the top of the effect's body, since it's accessed several times? I just wondered if it would look a little clearer.

```javascript
const currentEditor = editorInstanceRef.current!;
```